### ISSUE TITLE
Show Hunting tooltip when mousing over in World Map

### ISSUE THREAD

**Issue by nlruk** [runelite/runelite#11458]
Mining spots have specific ores in the world map when you hover over it

<img width="192" alt="mining" src="https://user-images.githubusercontent.com/63312904/80975036-7d8de800-8e19-11ea-8b82-32dd9e04341f.png">

Can you please add the specific type of hunting areas? Once this is done all the areas have been covered!!!!

<img width="378" alt="Screenshot 2020-05-04 at 15 13 20" src="https://user-images.githubusercontent.com/63312904/80975324-deb5bb80-8e19-11ea-8731-03706436df2d.png">

<img width="802" alt="Screenshot 2020-05-04 at 15 14 39" src="https://user-images.githubusercontent.com/63312904/80975444-0147d480-8e1a-11ea-84a5-31c1bd53e5f1.png">
<img width="601" alt="Screenshot 2020-05-04 at 15 15 18" src="https://user-images.githubusercontent.com/63312904/80975554-21779380-8e1a-11ea-861e-cdaf136d46c1.png">

**Comment by dekvall** [comment]
Duplicate of #6848
